schemas/system/Permission.schema

```diff
@@ -1,14 +1,12 @@
 System({
   Role: { category: 'Role', required: true, index: true },
   Category: { category: 'Category', required: true },
-  Catalog: { category: 'Catalog' },
-  Division: { category: 'Division' },
   Access: { domain: 'UserAccess', required: true },
 
-  NaturalKey: Index('Role', 'Category'),
+  // Catalog, Subdivision, Action
+  Target: { category: 'Identifier', required: true },
```

Comment:
What does the `Target` mean in terms of permissions?

Reply:
`Target` will store `Identifier` of one of the listed categories: `Catalog, Subdivision, Action`

Reply:
Okay, but I still don't understand what it means in terms of permissions.

Does it specify who is given the permissions specified in the `Access` field (I don't know what's the point of the `Role` field then) or is it for describing the set of objects to which permission is given to (what's the point of the `Category` field then)?

Reply:
Category identifies table, Target identifies either Catalog or Subdivision

Reply:
Target should be definitely renamed to something